src/core/engines/Cesium/Feature/Resource/index.tsx

```diff
@@ -132,6 +130,11 @@ export default function Resource({
     [updateClock, viewer?.clock],
   );
 
+  // convert hexCodeColorString to ColorValue?s
+  const strokeValue = stroke ? Color.fromCssColorString(stroke) : undefined;
+  const fillValue = fill ? Color.fromCssColorString(fill) : undefined;
+  const markerColorValue = markerColor ? Color.fromCssColorString(markerColor) : undefined;
```

Comment:
I think it's better to use `useMemo` because `Color.fromCssColorString` is a heavy task

Reply:
Done.